test/test_cuda/test_moe_model.py

```diff
@@ -103,3 +107,86 @@ def test_llama4(setup_llama4):
 
     # clean the output directory after test
     shutil.rmtree(output_dir, ignore_errors=True)
+
+
+class NewQwen3MLP(ReplacementModuleBase):
+    def __init__(self, original: Qwen3MLP, config: Qwen3Config):
+        super().__init__()
+        self.config = config
+        self.hidden_size = config.hidden_size
+        self.intermediate_size = config.intermediate_size
+        self.new_gate_proj = original.gate_proj
+        self.new_up_proj = original.up_proj
+        self.new_down_proj = original.down_proj
+        self.act_fn = nn.GELU()
+
+    def forward(self, x):
+        gate = self.new_gate_proj(x)
```

Comment:
I remember that there is a warning to remind user that transformers could run these models anymore. 
In this pr, this has been supported. Have we deleted this warning? please have a double check

Reply:
I’m not aware of the background, could you point me to the code? thanks